release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1449,41 +1447,55 @@ In the following tables, features are marked with the following statuses:
 |Technology Preview
 |Technology Preview
 
-|Selectable Cluster Inventory
+|Azure Tagging
+|Not Available
 |Technology Preview
 |Technology Preview
+
+|Enabling NIC partitioning for SR-IOV devices
+|Not Available
+|Technology Preview
 |Technology Preview
 
-|Multi-architecture compute machines
+|GCP Confidential VMs
+|Not Available
 |Technology Preview
 |General Availability
-|General Availability
 
-|Mount shared entitlements in BuildConfigs in RHEL
-|Technology Preview
-|Technology Preview
+|GCP Tagging
```

Comment:
Suggest to replace "GCP Tagging" with "User-defined labels and tags for Google Cloud Platform", thanks!

Reply:
Thanks! Updated.